### ISSUE TITLE
Flaying Knife stacking multiple bleeds

### ISSUE THREAD

**Issue by RonJomero** [foundryvtt/pf2e#6664]
The item Flaying Knife has a 1d8 Wounding effect.  But my party is consistently seeing it add 1d6+1d8 bleeding damage in Foundry.  I don't know if the system is confused that Wounding (which is normally 1d6) has changed, or if it's from the dagger specialization on a crit where it adds a bleed.   Regardless, as far as I know bleeds should not stack and instead take the higher of the two damages.

**Comment by stwlam** [comment]
There are no stacking restrictions on bleed or any other type of damage.

**Comment by RonJomero** [comment]
From AoN:

"You can be simultaneously affected by multiple persistent damage
conditions *so long as they have different damage types.* *If you would
gain more than one persistent damage condition with the same damage type,
the higher amount of damage overrides the lower amount.* The damage you
take from persistent damage occurs all at once, so if something triggers
when you take damage, it triggers only once; for example, if you're dying
with several types of persistent damage, the persistent damage increases
your dying condition only once."

**Comment by RonJomero** [comment]
Currently the Flaying Knife is stacking a 1d6 + 1d8 bleed on the target.  When it should be ignoring the 1d6 bleed and only doing the 1d8.

**Comment by stwlam** [comment]
Reopening this: while it's certainly clear that a creature can only have one persistent-damage condition of the same damage type,  less clear is whether bleed damage coming from different sources but in a single attack are added together normally like other damage types to create a single condition. We're seeking clarification from Paizo, which may never happen.

**Comment by KaruiKage** [comment]
I agree that these should not stack. "Persistent Damage ([Anything] X)" is a Condition, with a value of X. Conditions never stack. The condition happens to cause damage to be dealt at the end of a player's turn, but that is damage as the result of the condition, they are not the same thing.

**Comment by dharkus5000** [comment]
flaming and ashen add together too, they should definitely not
currently the ashen & flaming rune's persistent fire dmg stack in the system

they should not stack i'm pretty sure 
Ashen
>  A creature hit by an attack from an ashen weapon becomes surrounded by burning ash, which deals 1d4 persistent fire damage

Flaming
>  The weapon deals an additional 1d6 fire damage on a successful Strike, plus 1d10 persistent fire damage on a critical hit.
currently on a crit the persistent adds together in the dmg
Greater versions basically identical but higher dmg
